2.15/2.15.7/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.15.7
+В самом начале выводится 2 пары сообщений:
+1) "Введите имя" и "Введите возраст"
+2) "Введите имя" и "Введите возраст"
+
+А потом вывести 2 алерта с инфой из введенных данных:
+"Этого человека зовут - Х. Он [еще молод], потому что ему Y лет."
+Причем фраза [еще молод] может поменяться на [уже взрослый], если введенный для него возраст был больше 18.
+<script>
+	function showUserInformation(name, age) {
+		let ageText = 'еще молод';
+
+		ageText = (age > 18) ? 'уже взрослый' : ageText;
```

Comment:
you can simplify here as well, just declare the variable `ageText` here and then you will understand